### PR TITLE
Add deb installation to readthedocs

### PR DESCRIPTION
**Pull request by engelmi**
This PR updates the readthedocs to describe how to install BlueChi on Debian, which was recently added in #918. The `.deb` packages for BlueChi v0.8.0 (and upcoming versions) are now being hosted on GitHub in [eclipse-bluechi/bluechi-ppa](https://github.com/eclipse-bluechi/bluechi-ppa/).

### CODE REVIEW

File: doc/docs/getting_started/installation.md [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+
+```bash
+dnf install bluechi-controller \
+    bluechi-agent \
+    bluechi-ctl \
+    bluechi-selinux \
+    python3-bluechi
+```
+
+## Ubuntu or any Debian
```
Comment (mwperina) [inline doc/docs/getting_started/installation.md]: Wouldn't "Ubuntu or Debian" be enough (without "any")?

**Review by mwperina — APPROVED** [review]
Small nit, otherwise it looks great

File: doc/docs/getting_started/installation.md (hunk 1 above)
Comment (engelmi) [reply doc/docs/getting_started/installation.md]: How about `Debian-based distributions`?

Comment (mwperina) [reply doc/docs/getting_started/installation.md]: :+1:

**Review by mwperina — APPROVED** [review]

**Comment by engelmi** [comment]
Seems like there are problems on testing farm. 
PR only affects docs and the integration tests in the GitHub CI are passing, so lets merge this.

**Pull request merged**